tools/measurement/apksize/apksize.gradle

```diff
@@ -21,6 +21,10 @@ android {
     flavorDimensions "apkSize"
 }
 
+dependencies {
+    implementation("com.google.android.gms:play-services-basement:16.1.0")
```

Comment:
Why is this needed?

Reply:
Removed.

We used to see below failure from coverage tasks log (https://pantheon.corp.google.com/storage/browser/fireescape/pr-logs/pull/firebase_firebase-android-sdk/489/coverage-metrics-upload/1135971121242312705/?project=fireescape-c4819&folder=true&organizationId=true).
```
Task :tools:measurement:apksize:preFirestoreAggressiveBuild FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tools:measurement:apksize:preFirestoreAggressiveBuild'.
> Android dependency 'com.google.android.gms:play-services-basement' has different version for the compile (16.0.1) and runtime (16.1.0) classpath. You should manually set the same version via DependencyResolution
```

But looks like #489 fixed it, somehow.